docs/modules/migrate/pages/lts.adoc

```diff
@@ -0,0 +1,128 @@
+= Long-term Support Releases
+
+:description: Hazelcast simplifies your upgrade experience with the introduction of long-term support (LTS) releases. You can upgrade directly from a supported previous release to the LTS release or directly between LTS releases using a rolling upgrade.
+:page-enterprise: true
+
+{description}
+
+LTS releases have been introduced with 5.5. You can upgrade directly from 5.2 or later to 5.5 without the need to upgrade to each release between your current version and the LTS release.
```

Comment:
Should we specify that they should upgrade from the latest patch of each release line? We only test latest patch (at the time of the release).